pgpro_tam/README.md

```diff
@@ -0,0 +1,22 @@
+###pgpro_tam is an extensible table access method for vanilla Postgres
```

Comment:
Minor: let's remove the leading # ... that's for headlines

Reply:
Fixed